PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,19 @@
+## Issue
```

Comment:
I think this is a fair addition to the repository! Thanks. We need to take more care when to add details when we raise PRs.